### PR TITLE
Add Utapi account-level metrics

### PR DESCRIPTION
**Pull request by bennettbuchanan**
Pushes account-level metrics for any API that includes the `AuthInfo` object.
Depends on https://github.com/scality/utapi/pull/74

### CODE REVIEW

**Comment by bennettbuchanan** [comment]
SCALITY_UTAPI_BRANCH=ft/add_account_level_metrics
SCALITY_INTEGRATION_BRANCH=test/utapiAccountMetrics
@ironman-machine try

**Comment by bennettbuchanan** [comment]
SCALITY_UTAPI_BRANCH=ft/add_account_level_metrics
SCALITY_INTEGRATION_BRANCH=test/utapiAccountMetrics
@ironman-machine try

**Comment by bennettbuchanan** [comment]
SCALITY_UTAPI_BRANCH=ft/refactor_utapiClient_account_metric
@ironman-machine try

File: lib/utapi/utilities.js [hunk 1]
```diff
@@ -126,16 +126,26 @@ export function listBucketMetrics() {
  * @param {object} log - werelogs logger
  * @param {object} metricObj - the object containing the relevant data for
  * pushing metrics in Utapi
- * @param {string} metricObj.bucket - bucket name
- * @param {string} metricObj.byteLength - [optional] current object size (used,
+ * @param {string} metricObj.bucket - (optional) bucket name
```
Comment (electrachong) [inline lib/utapi/utilities.js]: you can put in brackets around `metricObj.bucket` to indicate it's optional
http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values

same for the other optional params

Comment (bennettbuchanan) [inline lib/utapi/utilities.js]: Updated.

**Comment by bennettbuchanan** [comment]
The website endpoints (`websiteGet`, `websiteHead`) will track bucket-level metrics, but it sounds like it does not make sense to track account-level because the API can be called by someone without an account.

**Comment by bennettbuchanan** [comment]
SCALITY_UTAPI_BRANCH=ft/refactor_utapiClient_account_metric
@ironman-machine try

**Comment by bennettbuchanan** [comment]
@ironman-machine

**Comment by bennettbuchanan** [comment]
@ironman-machine

**Comment by bennettbuchanan** [comment]
@ironman-machine try

**Comment by bennettbuchanan** [comment]
@ironman-machine try

**Comment by bennettbuchanan** [comment]
@ironman-machine try

**Comment by bennettbuchanan** [comment]
end-to-end passed

**Pull request merged**